libraries/RW/Utils/utils.py

```diff
@@ -5,6 +5,7 @@
 """
 from typing import Iterable, Any, Union, Optional
 import os, pprint, functools, time, json, datetime, yaml, logging, re, xml.dom.minidom, urllib.parse
+import jmespath
```

Comment:
add to requirements.txt?

Reply:
Good catch, missed this when creating a 'clean' commit from a messier fork.